<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Results</class>
 <widget class="QDialog" name="Results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_res">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Results</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_plag">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Plagiarised Lines</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_quote">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Quoted Lines</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ref">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Referenced Lines</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sim">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Similarity</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sim">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_quote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_ref">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_plag">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
